tests/phpunit/integration/api/term-meta/fixtures/test-fields.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ *  Test field data for use with render_fields method tests.
+ *
+ * @package Beans\Framework\Tests\Integration\API\Term_Meta\Fixtures
+ *
+ * @since 1.5.0
+ */
+
+return array(
+	'fields' => array(
+		array(
+			'id'      => 'beans_layout_test',
+			'label'   => 'Layout',
+			'type'    => 'radio',
+			'default' => 'default_fallback',
+			'options' => array(
+				'default_fallback' => 'Use Default Layout',
+				'c'                => BEANS_THEME_DIR . 'lib/admin/assets/images/layouts/c.png',
```

Comment:
Let's use the URL instead here.  Watch out to make sure you have a trailing slash between the URI and the rest of the pathing.